data/quarg/quarg news.txt

```diff
@@ -442,3 +442,65 @@ news "student on quarg planets"
 			"."
 		word
 			`"`
+
+news "quarg anywhere"
+	location
+		government "Quarg"
+	name
+		word
+			"Quarg"
+	message
+		word
+			`"Yes human? Do you have any questions?"`
+			`"Hello human. Are you enjoying your time on our world? We are glad to receive visitors."`
+			`"I am sorry, but we will not provide you with our technology. It is for the best that humans grow and develop on their own."`
+			`"It troubles me greatly to see how many species still cling to senseless fighting. I can only hope you will all come to see reason."`
+			`"Last year our scientific efforts were quite fruitful, but I am afraid we cannot simply share our discoveries with you."`
+			`"If you ever observe some piece of our technology in the possession of someone else, please report it to us. We cannot allow our equipment to be in the wrong hands."`
+			`"You are welcome to be amongst us here, provided you seek no conflict with anyone on our worlds."`
+			`There's an eerie silence as a particularly large crowd of Quarg walks by you.`
+			`A group of Quarg is enthralled in what seems an engaging discussion, until one more Quarg comes to them. They all turn to it, and after it says something quickly they all get up in unison and move to somewhere else in the hangars.`
+			`An older Quarg is playing with a much younger Quarg, young enough that it's still shorter than yourself. The strange game consists of passing to one another a jar with some slimy, metallic substance inside. Every time the jar is passed, the substance meshes itself into a different shape, sometimes into shapes you know not the meaning of.`
```

Comment:
```suggestion
			`An older Quarg is playing with a much younger Quarg, young enough that they're still shorter than yourself. The strange game consists of passing to one another a jar with some slimy, metallic substance inside. Every time the jar is passed, the substance meshes itself into a different shape, sometimes into shapes you know not the meaning of.`
```
This isnt my area of course, but still isnt it better this way? Or did we not get to the quarg with this issue yet? They're obviously sentient

Reply:
It's intentional that the Quarg pronouns are it/its, yes.

Reply:
that's weird but ok then we shouldnt change it in this PR